site/docs/install-windows.md

```diff
@@ -66,6 +67,16 @@ See [Chocolatey installation and package maintenance
 guide](https://bazel.build/windows-chocolatey-maintenance.html) for more
 information about the Chocolatey package.
 
+#### Install using Scoop
```

Comment:
Cool! Do you have any information about maintenance as well? E.g. how does one update the package after a Bazel release?

Also, do you plan to own this package? If so, could you please add info about that here, so if there are questions about the Scoop package, people know whom to ask?

Reply:
ping

Reply:
@laszlocsomor Yes, I periodically update Bazel version in the Scoop. What kind of contact would be okay? E-mail? Telegram?

Reply:
Great! Yes, whichever works best for you. :)

Reply:
I've made an instruction. Can you please check if everything is fine?
https://github.com/bazelbuild/bazel-website/pull/173

Reply:
LGTM. Thanks again!
I'll import this PR now.